app/views/application/cycle/edit/cycle.njk

```diff
@@ -20,20 +20,20 @@
           name: "applicatoncycle",
           fieldset: {
             legend: {
-              text: "Change cycle",
+              text: "Defer offer to next cycle",
```

Comment:
This no longer works when switching back to the previous cycle.

Reply:
OK

Reply:
Are you saying that on this page, you might use the green button to revert back to the old cycle? Sorry don't understand
![Screenshot 2020-07-08 at 14 49 01](https://user-images.githubusercontent.com/38726669/86926698-4a721880-c12a-11ea-9dea-f46e7f0eb7c9.png)